### PR TITLE
deepakdhakar

### PR DESCRIPTION
**Pull request by deepakdhakar17**
## Check List (Check all the applicable boxes) 

- [ x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6493"></a>

### CODE REVIEW

**Review by SaraJaoude — CHANGES_REQUESTED** [review]
Your file name needs to match your GitHub username. So instead of `data/deepakdhakar27.json` this should be `data/deepakdhakar17.json`

I would suggest you look at our QuickStart Guide in Docs: https://linkfree.io/docs/quickstart to help you update this Pull Request so that we can get it merged.

Your JSON is also failing, check this on: https://jsonlint.com so that you can see how to correct it.

File: data/deepakdhakar27.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  "type": "personal",
+  "bio": "Techno india njr",
+  "links": [
+    {
+      "name": "Follow me on Github",
+      "url": "https://github.com/deepakdhakar17",
+      "icon": "FaTgithub"
+    }
+  ]
+  }
```
Comment (HassanTanveer) [inline data/deepakdhakar27.json:13]:
```suggestion
{
  "name": "deepakdhakar17",
  "type": "personal",
  "bio": "Techno india njr",
  "links": [
    {
      "name": "Follow me on Github",
      "url": "https://github.com/deepakdhakar17",
      "icon": "FaTgithub"
    }
  ]
}
```

**Review by HassanTanveer — CHANGES_REQUESTED** [review]
Currently your JSON is invalid. Please commit this suggestion to fix it.

**Comment by SaraJaoude** [comment]
We look forward to seeing your Profile on LinkFree, but as there is no recent activity from you to make the changes suggested I am going to close this pull request.

We hope you will raise a new pull request to create your Profile and when you do, you might want to take a look at the following Docs which will help with the suggested changes the maintainers have proposed:

Quickstart: https://linkfree.io/docs/quickstart

Editing your Profile: https://linkfree.io/docs/how-to-guides/editing

**Pull request closed without merging**